app/src/main/AndroidManifest.xml

```diff
@@ -20,7 +20,10 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
-        tools:ignore="AllowBackup">
+        tools:ignore="AllowBackup"
+        android:requestLegacyExternalStorage="true"
```

Comment:
What are you using this for?

Reply:
The new way of storing files to internal storage on newer Android devices is much different than devices running Android 10 or lower. This property enables the storage provider to be used the same way as older devices. In the app received files can be exported to internal storage more easily with this enabled as I didn't want to put too much effort into this.